feg/cloud/go/services/feg_relay/gw_to_feg_relay/tests/relay_test.go

```diff
@@ -113,6 +245,9 @@ func TestNHRouting(t *testing.T) {
 	// Register FeG's test Hello Server
 	feg_protos.RegisterHelloServer(srv.GrpcServer, &testHelloServer{})
 
+	// Register FeG's Central Session Controller Server
+	sessionProxy := &testCentralSessionController{resultChan: make(chan string, 2)}
```

Comment:
shouldnt you have to have two `testCentralSessionController` assure they are sent to the right one?

Reply:
@uri200 , might not need two testCentralSessionController. I could see [here]([url](https://github.com/magma/magma/blob/master/feg/cloud/go/services/feg_relay/gw_to_feg_relay/tests/relay_test.go#L107)) only one serving FeG Server is simulated and different gRPC Servers are registered to it. [Here](https://github.com/magma/magma/blob/master/feg/cloud/go/services/feg_relay/gw_to_feg_relay/tests/relay_test.go#L121), trying to bypass the request to simulated Feg Server. So added one testCentralSessionController, and in UpdateSession rpc definition, checking for RATType and  the response object is filled accordingly. This ensured that UpdateSession is triggered twice and the collective responses are verified